apps/web/src/components/docs/docs.const.ts

```diff
@@ -1,20 +1,14 @@
 import { ROUTES } from '../../constants/routes';
 
 export const PATHS: { [key in ROUTES]?: string } = {
-  [ROUTES.ACTIVITIES]: 'activity-feed/introduction',
-  [ROUTES.LAYOUT]: 'content-creation-design/layouts',
-  [ROUTES.CHANGES]: 'platform/environments',
-  [ROUTES.INTEGRATIONS]: 'channels-and-providers/integration-store',
-  [ROUTES.SUBSCRIBERS]: 'subscribers/subscribers',
-  [ROUTES.WORKFLOWS]: 'workflows/notification-workflows',
-  [ROUTES.TENANTS]: 'tenants/introduction',
-  [ROUTES.TRANSLATIONS]: 'content-creation-design/translations',
-  [ROUTES.STUDIO_ONBOARDING]: 'framework/quickstart',
-  [ROUTES.STUDIO_ONBOARDING_PREVIEW]: 'framework/concepts/workflows',
-  [ROUTES.STUDIO_FLOWS]: 'framework/concepts/workflows',
-  [ROUTES.STUDIO_FLOWS_VIEW]: 'framework/concepts/workflows',
-  [ROUTES.STUDIO_FLOWS_STEP_EDITOR]: 'framework/concepts/workflows',
-  [ROUTES.STUDIO_FLOWS_TEST]: 'framework/concepts/workflows',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Should we just consider deprecating the old ones for now?‏